### PR TITLE
Add option to adjust MTU of the TUN device

### PR DESCRIPTION
**Pull request by heikobornholdt**
This MR adds a channel option for adjusting the MTU size of the created TUN device.

Example:
```java
final Bootstrap b = new Bootstrap()
        .channel(TunChannel.class)
        .option(TunChannelOption.TUN_MTU, 1234)
        .group(new NioEventLoopGroup(1))
        .handler(new ChannelInitializer<>() {
            @Override
            protected void initChannel(final Channel ch) {
                ...
            }
        });
final Channel ch = b.bind(new TunAddress(name)).syncUninterruptibly().channel();
ch.closeFuture().syncUninterruptibly();
```

Currently, this option is not supported on Windows as the MTU size [is hard-coded into wintun](https://git.zx2c4.com/wintun/tree/api/wintun.h#n203). Modifying this file to our needs is not possible, as we then need to provide a new valid signature. Without a valid signature, Windows will not load the driver.

Windows users can adjust the MUT size with the following command: `netsh interface ipv4 set subinterface tun0 mtu=1234 store=active`

### CODE REVIEW

File: src/main/java/org/drasyl/channel/tun/DefaultTunChannelConfig.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        if (option == TUN_MTU) {
+            return (T) Integer.valueOf(getMtu());
+        }
+        return super.getOption(option);
+    }
+
+    @Override
+    public <T> boolean setOption(final ChannelOption<T> option, final T value) {
+        if (!super.setOption(option, value)) {
+            if (option == TUN_MTU) {
```
[the remark this answers is not in the document]
Comment (heikobornholdt) [reply src/main/java/org/drasyl/channel/tun/DefaultTunChannelConfig.java:51]: ignore

**Comment by KevinRoebert** [comment]
LGTM

**Comment by KevinRoebert** [comment]
Maybe the hint for Windows users can be added to the README

**Comment by KevinRoebert** [comment]
The new channel option should be added to the README example

**Review by KevinRoebert — APPROVED** [review]
LGTM

**Pull request merged**